easybuild/easyconfigs/m/medaka/medaka-1.4.3-foss-2020b.eb

```diff
@@ -0,0 +1,77 @@
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Danilo Gonzalez <danilo.gonzalez@hpcnow.com>
+# License::     GPL-v3.0
+
+easyblock = 'PythonBundle'
+
+name = 'medaka'
+version = '1.4.3'
+
+homepage = 'https://github.com/nanoporetech/medaka'
+description = "medaka is a tool to create a consensus sequence from nanopore sequencing data."
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+builddependencies = [('Autotools', '20200321')]
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('TensorFlow', '2.5.0'),
```

Comment:
The assumption here is that a sufficiently recent `TensorFlow` version should work, but it probably doesn't, see https://github.com/nanoporetech/medaka/blob/master/requirements.txt#L9 and https://github.com/nanoporetech/medaka/issues/323

So by means of exception, we should use `TensorFlow` 2.2.2 here (or at best, latest 2.2.x), unfortunately...

Reply:
See also #13854 (TensorFlow v2.2.3 with `foss/2020b`) and #13853 (medaka with `foss/2020b`)